### DISCUSSION TITLE
Comment from your Grader!

### DISCUSSION THREAD

**Discussion by SamyAbisaleh** [Jeannelialbedo/Interactive-Lab-Hub#2] [General]
Hi Jeanne, 

Really liked your work. The designs were really clean and really easy to follow. My one concern is how are you actually going to design and build this project? Some things to think about are the sensing aspect of the project, how can you determine the glucose levels? 

Best,
Samy

**Comment by Jeannelialbedo** [comment]
Hi Samy,

Thank you so much for your advice! They are really helpful. Regarding the sensor to determine the glucose levels, in medical treatments, the hospitals usually analyze the near-infrared spectrum of the glucose concentration to determine the glucose levels. So the easiest way would be to deploy an infrared sensor.

Best,
Jeanne

**Reply by SamyAbisaleh** [reply]
Okay, awesome. Do you know what to look for using an infrared sensor? Like what is the infrared sensor sending?
